python-package/README.rst

```diff
@@ -348,6 +348,21 @@ To check that a contribution to the package matches its style expectations, run
 
     bash .ci/lint-python-bash.sh
 
+To run the tests locally and compute test coverage, run the following from the root of the repo.
+
+.. code:: sh
+
+    cmake -B build -S .
+    cmake --build build --target _lightgbm -j4
```

Comment:
TBH, I don't like that these lines are not cross-platform. Maybe it's better to use a reference to our build documentation?

Reply:
Looking at this again, I agree with you... this should just refer to the previous docs on how to install the package. Especially because this project doesn't support editable installs, and so the test are always expected to be run on a fully-installed package.

Changed this in https://github.com/microsoft/LightGBM/pull/6995/commits/474296d2a214fe1f4d4abaeab3b108ccb20653b0